keras/wrappers/scikit_learn.py

```diff
@@ -6,6 +6,18 @@
 from ..utils.np_utils import to_categorical
 
 
+class Hideout(object):
```

Comment:
Could you achieve the same with a different class attribute under a different name? Is the nesting essential?